image/templates/helm/stackrox-central/internal/defaults.yaml.htpl

```diff
@@ -304,11 +304,11 @@ defaults:
                 operator: DoesNotExist
       resources:
         requests:
-          cpu: "200m"
-          memory: "3Gi"
+          cpu: "1000m"
+          memory: "4000Mi"
```

Comment:
makes sense to increase all these. We are very DB heavy, especially the Matcher. I'm mostly curious why all units are `Mi`